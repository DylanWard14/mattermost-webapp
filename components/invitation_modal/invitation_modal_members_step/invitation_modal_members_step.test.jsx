// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {shallow} from 'enzyme';

import {shallowWithIntl} from 'tests/helpers/intl-test-helper';

import InvitationModalMembersStep from './invitation_modal_members_step';

describe('components/invitation_modal/InvitationModalMembersStep', () => {
    test('should match the snapshot', () => {
        const wrapper = shallow(
            <InvitationModalMembersStep
                teamName='Test Team'
                currentTeamId='test-team-id'
                inviteId='123'
                searchProfiles={jest.fn()}
                emailInvitationsEnabled={true}
                onSubmit={jest.fn()}
                onEdit={jest.fn()}
            />,
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('should match the snapshot when email invitations are disabled', () => {
        const wrapper = shallow(
            <InvitationModalMembersStep
                teamName='Test Team'
                currentTeamId='test-team-id'
                inviteId='123'
                searchProfiles={jest.fn()}
                emailInvitationsEnabled={false}
                onSubmit={jest.fn()}
                onEdit={jest.fn()}
            />,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
