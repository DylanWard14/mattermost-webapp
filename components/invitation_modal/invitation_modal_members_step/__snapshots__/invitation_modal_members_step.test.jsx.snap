// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/invitation_modal/InvitationModalMembersStep should match the snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`components/invitation_modal/InvitationModalMembersStep should match the snapshot when email invitations are disabled 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
